modules/javafx.web/src/main/native/Source/WebCore/rendering/svg/RenderSVGResourceMasker.cpp

```diff
@@ -89,6 +89,8 @@ bool RenderSVGResourceMasker::applyResource(RenderElement& renderer, const Rende
 bool RenderSVGResourceMasker::drawContentIntoMaskImage(MaskerData* maskerData, ColorSpace colorSpace, RenderObject* object)
 {
     GraphicsContext& maskImageContext = maskerData->maskImage->context();
+    auto deviceScaleFactor = document().deviceScaleFactor();
+    maskImageContext.applyDeviceScaleFactor(deviceScaleFactor);
```

Comment:
For javafx port specific code in WebKit, it should be enclosed in `#if PLATFORM(JAVA)`. But I think that there should be a better approach as other ports don't require this scaling of `deviceScaleFactor`

Reply:
RenderSVGResourceMasker.cpp changes are removed now.